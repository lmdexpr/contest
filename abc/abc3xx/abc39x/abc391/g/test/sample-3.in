7 50
atcoder
