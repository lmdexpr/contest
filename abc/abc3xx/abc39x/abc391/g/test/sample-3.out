309810541 226923474 392073062 146769908 221445233 435648037 862664208 238437587
